sdk/tables/Azure.Data.Tables/tests/TablesRecordedTestSanitizer.cs

```diff
@@ -15,7 +15,8 @@ public override string SanitizeVariable(string variableName, string environmentV
         {
             return variableName switch
             {
-                TablesTestEnvironment.PrimaryKeyEnvironmentVariableName => string.Empty,
+                TablesTestEnvironment.PrimaryStorageKeyEnvironmentVariableName => string.Empty,
```

Comment:
If you are sanitizing the entire value chances are you don't need to record it.

``` C#
public string PrimaryCosmosAccountKey => GetOptionalVariable(PrimaryCosmosKeyEnvironmentVariableName) ?? string.Empty;
```

should work.

Reply:
I will defer this as it broke recorded tests in an unexpected way. I'll look into it later